include/util/coordinate_calculation.hpp

```diff
@@ -85,16 +85,6 @@ double bearing(const Coordinate first_coordinate, const Coordinate second_coordi
 // Get angle of line segment (A,C)->(C,B)
 double computeAngle(const Coordinate first, const Coordinate second, const Coordinate third);
 
-// find the center of a circle through three coordinates
```

Comment:
I am not sure whether we should throw away that code. Doesn't really hurt to still have it around (e.g. for turn angles/radius computations).

I would strongly prefer to keep it for cases like these.